live-examples/css-examples/transforms/meta.json

```diff
@@ -7,175 +7,196 @@
             "cssExampleSrc": "./live-examples/css-examples/transforms/transform3d.css",
             "exampleCode": "./live-examples/css-examples/transforms/function-translateZ.html",
             "fileName": "function-translateZ.html",
             "title": "CSS Demo: translateZ()",
             "type": "css"
+        },
+        "perspective": {
+            "cssExampleSrc": "./live-examples/css-examples/transforms/transform3d.css",
+            "exampleCode": "./live-examples/css-examples/transforms/perspective.html",
+            "fileName": "perspective.html",
+            "title": "CSS Demo: perspective",
+            "type": "css"
+        },
+        "perspectiveOrigin": {
+            "cssExampleSrc": "./live-examples/css-examples/transforms/perspective-origin.css",
+            "exampleCode": "./live-examples/css-examples/transforms/perspective-origin.html",
+            "fileName": "perspective-origin.html",
+            "title": "CSS Demo: perspective-origin",
+            "type": "css"
+        },
+        "rotate": {
+            "cssExampleSrc": "./live-examples/css-examples/transforms/rotate3d.css",
+            "exampleCode": "./live-examples/css-examples/transforms/rotate.html",
+            "fileName": "rotate.html",
+            "title": "CSS Demo: rotate",
+            "type": "css"
+        },
+        "scale": {
+            "cssExampleSrc": "./live-examples/css-examples/transforms/transform3d.css",
+            "exampleCode": "./live-examples/css-examples/transforms/scale.html",
+            "fileName": "scale.html",
+            "title": "CSS Demo: scale",
+            "type": "css"
+        },
+        "transform": {
+            "exampleCode": "./live-examples/css-examples/transforms/transform.html",
+            "fileName": "transform.html",
+            "title": "CSS Demo: transform",
+            "type": "css"
+        },
+        "transformOrigin": {
+            "cssExampleSrc": "./live-examples/css-examples/transforms/transform-origin.css",
+            "jsExampleSrc": "./live-examples/css-examples/transforms/transform-origin.js",
+            "exampleCode": "./live-examples/css-examples/transforms/transform-origin.html",
+            "fileName": "transform-origin.html",
+            "title": "CSS Demo: transform-origin",
+            "type": "css"
+        },
+        "transformStyle": {
+            "cssExampleSrc": "./live-examples/css-examples/transforms/transform-style.css",
+            "exampleCode": "./live-examples/css-examples/transforms/transform-style.html",
+            "fileName": "transform-style.html",
+            "title": "CSS Demo: transform-style",
+            "type": "css"
+        },
+        "translate": {
+            "cssExampleSrc": "./live-examples/css-examples/transforms/transform3d.css",
+            "exampleCode": "./live-examples/css-examples/transforms/translate.html",
+            "fileName": "translate.html",
+            "title": "CSS Demo: translate",
```

Comment:
Does it make sense to give it the title `CSS Demo: 3d translate`? Same for the other ones.

Reply:
In all CSS examples, the title contains just the name of a property or a function, so I think adding a "3D" prefix could be misleading.

Reply:
That's true. What about in the file name? We have `transform3d` above, what do you think about `translate3d.html` or `translate-3d.html` or similar?

Reply:
I don't quite understand. The property name is called [translate](https://developer.mozilla.org/en-US/docs/Web/CSS/translate), so we want to reference it by its name like that:
```
{{EmbedInteractiveExample("pages/css/translate.html")}}
```
while a function called [translate3d](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/translate3d) is called like that:
```
{{EmbedInteractiveExample("pages/css/function-translate3d.html")}}
```
Those 3 new properties might be similar to 3d functions, but they have differences too. They all allow less than 3 values to be placed and the syntax of the scale property is totally different from scale3d.

Reply:
FYI, I'm considering adding [something like this codepen](https://codepen.io/bsmth/pen/bGKzVzq) to the [examples section](https://developer.mozilla.org/en-US/docs/Web/CSS/translate#examples) as to show that you can do 2D transforms with regular elements without special handling.